README.md

```diff
@@ -23,3 +23,11 @@ Supported platforms:
 This repository contains C-language examples that link to the native library targets and perform basic rendering and computation. You can test them with `make`.
 
 The idiomatic Rust wrapper lives in https://github.com/gfx-rs/wgpu-rs and provides a number of more complex examples to get a feel of the API.
+
+To run the C triangle example, run these commands at the root of the repo:
+```
+rustup toolchain install nightly-x86_64-unknown-linux-gnu
```

Comment:
We shouldn't require a specific toolchain – this should work fine on macOS, for example

Reply:
Oops forgot to expand on that.

Reply:
I guess they need a c compiler as well....